browser/css/jsdialogs.css

```diff
@@ -2054,36 +2054,24 @@ kbd,
 }
 
 /* DocumentPropertiesDialog Custom props */
-#DocumentPropertiesDialog #customprops .jsdialog .ui-grid .menubutton {
-	line-height: 16px;
-}
-
-#DocumentPropertiesDialog #customprops div.jsdialog.ui-grid-cell,
-#DocumentPropertiesDialog #customprops .spinfieldcontainer,
-#DocumentPropertiesDialog #customprops input.ui-timefield {
-	margin-top: auto;
-	margin-bottom: auto;
-}
-
-#DocumentPropertiesDialog #customprops #properties > .ui-grid {
-	display: flex;
-}
-
-#DocumentPropertiesDialog #customprops #properties > .ui-grid > div:nth-child(1) {
-	width: 150px;
+#headerbox,
```

Comment:
can we add `#DocumentPropertiesDialog` at the beginning? then it will be obvious for which dialog it is, and also will prevent applying the rule somewhere else. these ids are not that very unique

Reply:
added, and also in the other cases